smoke-tests-otel-starter/spring-boot-3/src/main/java/io/opentelemetry/spring/smoketest/DatasourceConfig.java

```diff
@@ -6,15 +6,14 @@
 package io.opentelemetry.spring.smoketest;
 
 import io.opentelemetry.api.OpenTelemetry;
-import io.opentelemetry.instrumentation.jdbc.datasource.JdbcTelemetry;
 import javax.sql.DataSource;
 import org.apache.commons.dbcp2.BasicDataSource;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 
 @Configuration
-@Profile("!jdbc-driver-config")
+@Profile("user-has-defined-datasource-bean")
```

Comment:
:100: - it took me a while to figure out what that means

Reply:
It's to test the case where the user has defined a datasource bean. Any other idea to have something explicit?